Ginger/Ginger/ApplicationModelsLib/POMModels/POMEditPage.xaml.cs

```diff
@@ -165,13 +165,19 @@ private void FillTargetAppsComboBox()
                 {
                     mPOM.TargetApplicationKey = key;
                 }
-                else
-                {                    
-                    Reporter.ToUser(eUserMsgKey.MissingTargetApplication, "The mapped" + mPOM.Key.ItemName + "Target Application was not found, please select new Target Application");
-                }
             }
 
-            xTargetApplicationComboBox.ItemsSource = WorkSpace.Instance.Solution.ApplicationPlatforms.Where(x => ApplicationPOMModel.PomSupportedPlatforms.Contains(x.Platform)).ToList();
+            xTargetApplicationComboBox.ItemsSource = null;
+            var targetPlatform = WorkSpace.Instance.Solution.GetTargetApplicationPlatform(mPOM.TargetApplicationKey);
+            if (targetPlatform != ePlatformType.NA)
```

Comment:
also add check if Targetplatform != null

Reply:
null is not required here as function GetTargetApplicationPlatform(mPOM.TargetApplicationKey) never return null.